Bottom: 1b2d31fb667f4ba02adefbd6e5ecab99a7b12821
Top:    770d7c46fadff79c625b7f3f30e2a31277ff0cb4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:42:43 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
